mypy_django_plugin/transformers/managers.py

```diff
@@ -115,6 +118,22 @@ def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[P
     )
 
 
+def _find_type_var(queryset_info: TypeInfo, ret_type: TypeVarType) -> Optional[MypyType]:
+    """
+    Attempts to find the concrete type corresponding to a TypeVarType in a queryset's base types.
+
+    Example:
+    Suppose queryset_info is based on a generic class `QuerySet[T]` and ret_type corresponds
+    to `T`, then this function will return the concrete type that `T` was instantiated with.
+    """
+    for base in queryset_info.bases:
+        for i, type_var in enumerate(base.type.type_vars):
+            type_var_path = f"{base.type.module_name}.{type_var}"
```

Comment:
This limits the `TypeVar` to be declared in the same module as the base class, right? That could mismatch if someone imports their `TypeVar` I think?

Reply:
you are right, I found a way to make it cleaner  with `base.type.defn.type_vars` instead of `base.type.type_vars` in my last commit